src/mame/tvgames/xavix.cpp

```diff
@@ -2014,134 +2047,134 @@ void xavix_state::init_xavix()
 ***************************************************************************/
 
 ROM_START( taitons1 )
-	ROM_REGION( 0x200000, "bios", ROMREGION_ERASE00 )
+	ROM_REGION( 0x800000, "bios", ROMREGION_ERASE00 )
 	ROM_LOAD( "taitonostalgia1.u3", 0x000000, 0x200000, CRC(25bd8c67) SHA1(a109cd2da6aa4596e3ca3abd1afce2d0001a473f) )
 ROM_END
```

Comment:
The fact that you need to change every single one of these to pad the ROM region out to 8 MiB should indicate that you’re doing this in a very fragile way that creates a lot of extra work when something changes.  Can you think of a way to do this that doesn’t require doing stuff like this?